### PR TITLE
Add aws-glue-sessions and sagemaker-kernel-wrapper

### PR DESCRIPTION
**Pull request by claytonparnell**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: template/Dockerfile [hunk 1]
```diff
@@ -52,6 +52,17 @@ RUN micromamba install -y --name base --file /tmp/$ENV_IN_FILENAME && \
 ARG MAMBA_DOCKERFILE_ACTIVATE=1
 RUN sudo ln -s $(which python3) /usr/bin/python
 
+# Install glue kernels, and move to shared directory
+RUN install-glue-kernels && \
+    jupyter-kernelspec install /opt/conda/lib/python3.10/site-packages/aws_glue_interactive_sessions_kernel/glue_pyspark --user && \
```
Comment (just4brown) [inline template/Dockerfile:57]: We shouldn't hardcode a python version in this path from the template dockerfile, as that can change depending on the sagemaker-distribution major version. Is there an env variable or something we can use to dynamically resolve the path?

**Review by just4brown — CHANGES_REQUESTED** [review]

File: template/Dockerfile (hunk 1 above)
Comment (just4brown) [reply template/Dockerfile:57]: I think there's a similar use case in the analytics PR. Might be able to use same env var used there: https://github.com/aws/sagemaker-distribution/pull/90/files

Comment (claytonparnell) [reply template/Dockerfile]: Added dynamic site packages logic

**Review by just4brown — APPROVED** [review]

File: test/test_artifacts/v1/glue-sessions.test.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+ARG COSMOS_IMAGE
+FROM $COSMOS_IMAGE
+
+ARG MAMBA_DOCKERFILE_ACTIVATE=1
+
+COPY --chown=$MAMBA_USER:$MAMBA_USER aws-glue-sessions/run_glue_sessions_notebook.sh .
+RUN chmod +x run_glue_sessions_notebook.sh
+COPY --chown=$MAMBA_USER:$MAMBA_USER aws-glue-sessions/glue_notebook.ipynb .
+RUN chmod +x glue_notebook.ipynb
```
Comment (navinsoni) [inline test/test_artifacts/v1/glue-sessions.test.Dockerfile:9]: nit: we do not have to add executable permissions to ipynb

**Review by navinsoni — APPROVED** [review]

**Pull request merged**